rllib/tuned_examples/maml/ant-rand-goal-maml.yaml

```diff
@@ -0,0 +1,26 @@
+ant-rand-goal-maml:
+    env: ray.rllib.examples.env.ant_rand_goal.AntRandGoalEnv
+    run: MAML
+    stop:
+        training_iteration: 1001
```

Comment:
If these are actually learning, could you add some decent stopping criteria here? Like episode_reward_mean and training_iteration (or better timesteps_total).

Reply:
The episode_reward_mean is the pre adaptation return. The post adaptation return is episode_reward_mean_adapt_[#]. For timesteps_total, I think training_iteration is a better criteria b/c that is how many meta-update steps it took to reach a good meta-policy. (one metaupdate per iteration)